addons/website/views/website_templates.xml

```diff
@@ -68,18 +68,29 @@
     </xpath>
 </template>
 
+<template id="website.wysiwyg" name="Website Editor Assets (used in website editor)">
+    <t t-call-assets="website.wysiwyg_assets"/>
+</template>
+
+<template id="website.wysiwyg_assets" name="Website Editor Assets (used in website editor)">
+    <t t-call="web._assets_helpers"/>
+    <link rel="stylesheet" type="text/scss" href="/web_editor/static/src/scss/bootstrap_overridden.scss"/>
+    <link rel="stylesheet" type="text/scss" href="/web_editor/static/src/scss/wysiwyg_variables.scss"/>
```

Comment:
Check where variables and bootstrap overrides are done now, this will not lead to anything good ;)

Reply:
Here we need mixin who are in the helphers. The bootstrap_overwridden is specific to the editor, more precisely the colors only.

Reply:
Yes, then as I said the variables should be put inside the t-call not after it. Same as that kind of stuff:
- https://github.com/odoo/odoo/blob/master/addons/web/views/report_templates.xml#L8
- https://github.com/odoo/odoo/blob/master/addons/web/views/webclient_templates.xml#L143

Reply:
Well you should not :) Read the "Note" in this commit message: https://github.com/odoo/odoo/commit/6e4db7d13a926845bf82f5b035afede42d6d5a89

Reply:
Still this, put the variables files inside the `t-call="_assets_helpers"`

Reply:
Many things are broken in CSS (eg "Customize" button), this requires a big fix.

Reply:
@qsm-odoo I do not understand your commit, please show me what I need to do in scss files that must be modified from your comments. If not, I will have to merge and make the cosmetic changes you want.

Reply:
@Gorash 

> put the variables files inside the `t-call="_assets_helpers"`

>

> Same as that kind of stuff:
> 
>     * https://github.com/odoo/odoo/blob/master/addons/web/views/report_templates.xml#L8
> 
>     * https://github.com/odoo/odoo/blob/master/addons/web/views/webclient_templates.xml#L143

Not sure what more I could tell you...